tests/lib/rules/valid-jsdoc.js

```diff
@@ -429,6 +429,14 @@ ruleTester.run("valid-jsdoc", rule, {
                 type: "Block"
             }]
         },
+        {
```

Comment:
Sorry, maybe I'm missing something, but based on the issue this is fixing, `@returns void` shouldn't cause an error. As far as I can tell, @nzakas said that type is not required for `return` statement.